### DISCUSSION TITLE
Starting novelWriter in Windows doesn't work

### DISCUSSION THREAD

**Discussion by AnetteMM** [vkbo/novelWriter#979] [Help]
When I try to start novelWriter I get this message:
![image](https://user-images.githubusercontent.com/42245289/152149251-c93b73e2-43f9-4a75-bf45-d4d172be8a65.png)

How can I fix it?
Thank you for your assist.

**Comment by vkbo** [comment]
That's not a message, it seems you just opened the source file. How did you try to start novelWriter?

Did you go through the two steps here? https://novelwriter.readthedocs.io/en/latest/setup_windows.html

**Reply by AnetteMM** [reply]
Yes, I installed phyton, clicked "... path ..." and started one time via 
![image](https://user-images.githubusercontent.com/42245289/152163842-ed279c65-f091-4681-9715-5a3d0674df3e.png)
and one time via the desktop icon

**Reply by vkbo** [reply]
I don't know what the screenshot is from, but what happens if you just double click the novelWriter.pyw file inside the app folder?

Did you also run the windows_install.bat file?

**Reply by AnetteMM** [reply]
Yes, i ran the windows_install. I additionally installed pip, PyQt5 and Visual Studio c++, restarted and now the last red notice is 

![image](https://user-images.githubusercontent.com/42245289/152194919-79d55985-59a0-474a-9a8d-85852ebb1da5.png)


I tried my best to install these build tools, but this is what I get repeatedly when I run the window_install.

**Reply by vkbo** [reply]
There should be no need to install all that extra stuff, but it seems like the error is with PyQt5. I know there has been some issues with the PyQt5-sip package. Usually it's because of an older version of pip.

Could you try to run:
```
pip install --upgrade pip
```

and then run the windows_install.bat script again?

**Reply by vkbo** [reply]
I just uploaded a Windows setup.exe file for both release 1.5.5 and 1.6-beta1 on the Releases page.

They contain the Python 3.10.2 embedded executable, the novelWriter Python files compiled for 3.10, and the three library dependencies. Let me know if that works. I've only tested it on a Windows 10 virtual machine.

**Reply by peter88213** [reply]
@AnetteMM 
To me, it looks like Python scripts are not associated with the Python interpreter on your system.
Have you tried right-clicking the desktop icon and selecting _open with Python_?
